src/osx/core/bitmap.cpp

```diff
@@ -1004,7 +1004,7 @@ wxBitmap wxBitmap::GetSubBitmap(const wxRect &rect) const
     if ( GetBitmapData()->m_bitmapMask )
     {
         wxMemoryBuffer maskbuf ;
-        int rowBytes = GetBestBytesPerRow( destwidth * kMaskBytesPerPixel );
+        int rowBytes = static_cast<int>(GetBestBytesPerRow( destwidth * kMaskBytesPerPixel ));
```

Comment:
It looks like we could avoid the cast by just using

```suggestion
        size_t rowBytes = GetBestBytesPerRow( destwidth * kMaskBytesPerPixel );
```

instead (here and below).